### ISSUE TITLE
Bug in Debugger - TOPNSKIP

### ISSUE THREAD

**Issue by edhans** [TabularEditor/TabularEditor3#954]
**Describe the issue**
I was working through debugger on the portion of this measure. The var6MonthRange is a table of YYYYMM values - 6 of them - from the calendar table.
<img width="941" alt="image" src="https://github.com/TabularEditor/TabularEditor3/assets/45298358/d910df6b-90f0-41b0-a8d0-6bd230574d7b">
Once I got to the [Total Sales] part the Current Expression showed the value for the 201705 month (the first in the var6MonthRange. Then the arrows in the toolbar lit up to go forward and back rows. But when I clicked on the Next row, the above image happened. Everything showed no values, just the TOPNSKIP error message.

** Which version are you currently using? **
Tabular Editor 3.8, Enterprise against Power BI Desktop

**To Reproduce**
I am working this this basic query generated by a Matrix viz with the year and month name on the rows.
```
		SUMMARIZECOLUMNS(
			ROLLUPADDISSUBTOTAL(
				'Calendar'[Year], "IsGrandTotalRowTotal",
				ROLLUPGROUP('Calendar'[Month Name], 'Calendar'[Month Number]),
				"IsDM1Total",
				NONVISUAL(__DM3FilterTable)
			),
			"Total_Sales", 'Measures Table'[Total Sales],
			"Rolling_6_Months_Sales", 'Measures Table'[Rolling 6 Months Sales]
		)
```

**Comment by edhans** [comment]
Note: I also created this in the Pivot Grid:
No columns
Year and Month Name in the Rows, Total Sales and this measure [Rolling 6 Month Average] in the values. Same result when iterating through the rows of AVERAGEX. TOPNSKIP error

**Comment by DBojsen** [comment]
Hi @edhans 

I have added this as a bug on our backlog, thanks for reporting.

Best Regards

David - Tabular Editor 3 Support

**Comment by otykier** [comment]
Fixed in [3.11.0](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.11.0). Thanks for reporting!
